projects/plugins/crm/includes/ZeroBSCRM.FormatHelpers.php

```diff
@@ -850,6 +850,44 @@ function zeroBSCRM_html_quoteStatusLabel($quote=array()){
 
 	}
 
+/**
```

Comment:
I believe this function will result in timezone issues...I'm extremely paranoid about those, as I think I successfully wiped (most of) them out over the past few years and would hate to introduce more. I'll have to look again tomorrow, but:

* Some fields are stored in `YYYY-MM-DD` format, and to properly print in WP format we'd need to make sure we are not adding/removing an offset that could put us in a different day.
* Some fields are stored in UTS, which means we have to add the proper offset before retrieving.

I'm struggling to read this function (lots of variable soup). Will look with fresher eyes tomorrow.

Also, I'm not quite sure why we're making this quote-specific.

Reply:
Yes, you were right re timezones (and good point about being quote specific). I've addressed both of those now.